ShopperInsights/src/main/java/com/braintreepayments/api/shopperinsights/v2/ShopperInsightsClientV2.kt

```diff
@@ -70,6 +72,46 @@ class ShopperInsightsClientV2 internal constructor(
         }
     }
 
+    /**
+     * Updates an existing customer session.
+     *
+     * @param customerSessionRequest: a [CustomerSessionRequest] object containing the request parameters
+     * @param sessionId: the ID of the session to update
+     * @param customerSessionCallback: a callback that returns the result of the customer session update
+     *
+     * Note: **This feature is in beta. Its public API may change in future releases.**
+     */
+    fun updateCustomerSession(
+        customerSessionRequest: CustomerSessionRequest,
+        sessionId: String,
+        customerSessionCallback: (customerSessionResult: CustomerSessionResult) -> Unit
+    ) {
+        updateCustomerSessionApi.execute(customerSessionRequest, sessionId) { result ->
+            when (result) {
+                is UpdateCustomerSessionApi.UpdateCustomerSessionResult.Success ->
+                    customerSessionCallback(CustomerSessionResult.Success(result.sessionId))
+                is UpdateCustomerSessionApi.UpdateCustomerSessionResult.Error ->
+                    customerSessionCallback(CustomerSessionResult.Failure(result.error))
+            }
+        }
+    }
+    /**
+     * Represents the result of a customer session operation (create or update).
+     */
+    sealed class CustomerSessionResult {
```

Comment:
It looks like this already exists: https://github.com/braintree/braintree_android/blob/payment-ready-v2/ShopperInsights/src/main/java/com/braintreepayments/api/shopperinsights/v2/CustomerSessionResult.kt

I don't think we need to duplicate. Should be able to cleanup the test file and revert some of the changes there from adding this.